languages/java/oso/src/main/java/com/osohq/oso/Exceptions.java

```diff
@@ -225,8 +225,8 @@ public InstantiationError(String className, Exception e) {
     }
 
     public static class InlineQueryFailedError extends PolarRuntimeException {
-        public InlineQueryFailedError() {
-            super(null, null);
+        public InlineQueryFailedError(String source) {
+            super("Inline Query Failed " + source);
```

Comment:
I think it would be worth standardizing this error messages across the libs. I know we need a general standardization pass that goes beyond this single error type, but no reason not to go for incremental improvement.